mitmproxy/addons/export.py

```diff
@@ -106,6 +107,69 @@ def httpie_command(f: flow.Flow) -> str:
     return cmd
 
 
+def python_requests_command(f: flow.Flow) -> str:
+    # Copy as Python Requests
+    request = cleanup_request(f)
+    request = pop_headers(request)
+
+    def escape_quotes(s: str) -> str:
+        return s.replace('"', '\\"')
+
+    code = "import requests\n\n"
+
+    code += f'url = "{request.pretty_url}"\n'
+    code += "headers = {\n"
+    cookies = ""
```

Comment:
Can you please use https://docs.mitmproxy.org/stable/api/mitmproxy/http.html#Request.cookies to construct the cookies attribute?